app/src/androidTest/java/org/simple/clinic/user/UserSessionAndroidTest.kt

```diff
@@ -131,7 +134,7 @@ class UserSessionAndroidTest {
         .saveOngoingLoginEntry(ongoingLoginEntry)
         .andThen(userSession.loginWithOtp(testData.qaUserOtp()))
         .toCompletable()
-        .andThen(userSession.logout())
+        .andThen(userSession.logout().toCompletable())
```

Comment:
Can we simplify this to
```
        .andThen(userSession.loginWithOtp(testData.qaUserOtp()))
        .flatMap { userSession.logout() }
        .blockingGet()
```
And similarly in all the places where we are using `.toCompletable()` throughout this file.

Reply:
Done.